frontend/src/stories/Structure-DatePicker.mdx

```diff
@@ -0,0 +1,80 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Structure/Date picker" />
+
+# Structure: Date picker
+
+## Design
+
+### What it is and when to use it
+
+- Date picker is used to allow users to input dates visually (by clicking on a calendar sheet)
+- Can be attached to input fields (drop modal) or embedded in modals (Baseline, work package date picker)
+
+### Variations
+
+None. (Removed header?)
+
+### Options
+
+- Working days only (non-working days are not clickable)
+- All days (working days are indicated but remain clickable)
+
+### States
+
+- Selected (single date)
+	- All corners rounded
+- Selected (date range)
+	- Start/finish: one side rounded
+	- Spanned dates: highlighted
+- Today
+- Non-working days
+- Disabled: all of the above states can be disabled, which will still indicate all these states but in shades of grey.
+
+### Design notes
+
+- Date format depends on user / instance settings
+- Certain dates can be disabled by other functions (relations)
+
+### Accessibility
+
+- Contrast ratios respected
+- Individual date selection not possible via keyboard navigation, known Flatpickr issue
+	- If selectable, label ("13") is not enough; screen reader should read the full date, "Friday, 13 October, 2023")
+
+## Angular components
```

Comment:
@psatyal As I understand Storybook now, I think that every Angular Component should have its own mdx file. And each variations of an Angular component shall have their own story. In other words, I don't think that it will work to have all Angular components listed in one mdx file as sketched here. @b12f correct?

Reply:
You can have as many `mdx` files as you like; they're just markdown files. It also sometimes makes sense to explain multiple angular components in one place. For example, both the single date picker and range date picker have their own components, but it makes sense to document them and show their stories in the same mdx.

Reply:
@b12f Oh, I see. So the (generated) documentation for inputs and outputs (the "Controls" tab) is per story and independent from mdx file?

Reply:
The documentation for inputs and outputs is generated by using the `ArgsTable` react component inside of an `mdx` file and giving it a reference to the angular component class it should be showing the args for: https://github.com/opf/openproject/blob/dev/frontend/src/app/spot/components/checkbox/stories/Checkbox.mdx?plain=1#L18